Makefile

```diff
@@ -26,8 +26,11 @@ VELOX_ENABLE_DUCKDB ?= "ON"
 
 CMAKE_FLAGS := -DTREAT_WARNINGS_AS_ERRORS=${TREAT_WARNINGS_AS_ERRORS}
 CMAKE_FLAGS += -DENABLE_ALL_WARNINGS=${ENABLE_WALL}
-CMAKE_FLAGS += -DVELOX_BUILD_TESTING=$(VELOX_BUILD_TESTING)
-CMAKE_FLAGS += -DVELOX_ENABLE_DUCKDB=$(VELOX_ENABLE_DUCKDB)
+
+#CMAKE_FLAGS += -DVELOX_DEFAULT=ON
+CMAKE_FLAGS += -DVELOX_BUILD_TESTING=${VELOX_BUILD_TESTING}
+CMAKE_FLAGS += -DVELOX_ENABLE_DUCKDB=${VELOX_ENABLE_DUCKDB}
```

Comment:
don't we need to hook up the env variables with correct -D flag in here as well?

Reply:
sure, will add those. somehow I added them during the development but got deleted during my final cleanup.